src/posts_storage.py

```diff
@@ -24,25 +24,27 @@ def __init__(self, session_maker) -> None:
         logger.info("Post storage is initializing...")
         self.session_maker = session_maker
 
-    def post(self, message: Message) -> None:
+    def post(self, message: Message, channel_name: str) -> None:
         original_message_id = message.fwd_from.channel_post
         original_peer_id = get_peer_id(message.fwd_from.from_id)
+
+        logger.info(f"Adding message {original_message_id} from channel '{channel_name}' (ID: {original_peer_id})")
```

Comment:
It is a bad practice to use f-string with logging. And I suppose with loguru too
https://docs.python.org/3/howto/logging.html#optimization

Also this line is a duplicate for the next line. Can you delete the next line, please

Reply:
I deleted mine in fact, I needed this log for the task to check the logs to see if I'm transmitting everything correctly.